server/src/api/merchants/index.ts

```diff
@@ -20,10 +20,13 @@
  */
 
 import express from 'express';
+import {merchantsService} from 'services';
+
 const router: express.Router = express.Router();
 
-router.get('/', (req, res, next) => {
-  res.send('merchants');
+router.get('/', async (req, res, next) => {
+  const merchants = (await merchantsService.getMerchants())[0];
```

Comment:
Why it only returns [0]? What's other data?

Reply:
The results of `getMerchants` contains an array, where the first element is the result, and the second element is an object related to the db query. An example is the following:

`[ [ { name: 'sampleMerchant',
      email: 'sampleMerchant@gmail.com',
      password: 'samplePassword',
      vpa: 'sample@vpa',
      [Symbol(KEY)]: [Key] },
    { email: 'sampleMerchant@gmail.com',
      password: 'samplePassword',
      vpa: 'sample@vpa',
      name: 'sampleMerchant',
      [Symbol(KEY)]: [Key] }],
  { moreResults: 'NO_MORE_RESULTS',
    endCursor:
'CkwSRmoQYn5ncGF5LWdyb3VwLWJ1eXIyCxIITWVyY2hhbnQiJGRjOTdlMjIwLWM0MjItOTNkZi1jOTI3LTZjOWVlNDMxOGY0MwwYACAA' } ]`

But on second thought, this unpacking should have been done in the model layer and not here.